app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.recent_posts
+  end
```

Comment:
- [ ] Kindly remove this function from here because it is making it difficult to see all the posts for a user. This method should only be used in views to control the flow of data. When I clicked the `See all posts`, I only saw three recent posts.

Reply:
Fixed! 
✅ `See all posts`: you can see all the posts now, and only the 3 more recents.
✅ `User name`: you can see the 3 more recent comments when you click on the user name.